src/ResourceManager/DataFactories/AzureRM.DataFactories.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '3.1.0'
+ModuleVersion = '3.2.0'
```

Comment:
@nicolaszhang please revert the change made to the module version; we will update the version during the release process